src/experimentation-ess/common.js

```diff
@@ -348,11 +351,67 @@ function mergeData(experiment, experimentMetadata, url) {
   return experiment;
 }
 
-function addPValues(experiment) {
-  for (const variant of experiment.variants) {
-    variant.p_value = 'coming soon';
+async function invokeLambdaFunction(payload) {
+  const lambdaClient = new LambdaClient({
+    region: 'us-east-1',
+    credentials: defaultProvider(),
+  });
+  const invokeParams = {
+    FunctionName: SPACECAT_STATISTICS_SERVICE_ARN,
+    InvocationType: 'RequestResponse',
+    Payload: JSON.stringify(payload),
+  };
+  const response = await lambdaClient.send(new InvokeCommand(invokeParams));
+  return JSON.parse(new TextDecoder().decode(response.Payload));
+}
+
+async function addPValues(experimentData) {
+  const lambdaPayload = {
+    type: 'statsig',
+    payload: {
+      rumData: {
+      },
+    },
+  };
+  for (const experiment of experimentData) {
+    lambdaPayload.payload.rumData[experiment.id] = {};
+    const metric = experiment.conversionEventName || 'click';
```

Comment:
do we take into the conversion event value into account here? If the conversion is defined as clicks on `.specific-block` or event a `convert` event with a specific source?

Reply:
No, custom conversion events with source and selectors, may be we can do it later